### DISCUSSION TITLE
Change Breadcrumb when using NavigationStore.NavigateExternal

### DISCUSSION THREAD

**Discussion by DerTomDerTwitch** [lepoco/wpfui#363] [Q&A]
Hello everyone,

I want to open a dynamic page from another page that is included in RootNavigation. I use `NavigationStore.NavigateExternal(page)` which works fine so far. But the Breadcrumb control still keeps the name of the original page. Any chance to change this or how to handle a case like this correctly?

In summary:
Page A (included in RootNavigation). When navigated to, the Breadcrumb display "A". On this page is a button to page B (not included in RootNavigation). Page B opens, but the display remains on "A".

**Comment by pomianowski** [comment]
This is not a discussion, please open an Issue for your problem.
